app/views/environments/welcome.html.erb

```diff
@@ -11,4 +11,8 @@
   <div class="blank-slate-pf-main-action">
       <%= new_link(_('New Puppet environment'), {}, { :class => "btn-lg" }) %>
   </div>
+  <p><%= _('Or') %></p>
```

Comment:
how strict should we follow it? according to the PF, it should be separate button for each proxy but I'd prefer having the select (current implementation) since there might be a lot of proxies available...